README.dk.md

```diff
@@ -1,63 +1,75 @@
 
-Udgivelser er signeret med [E88F5E48](https://keys.openpgp.org/search?q=34FF9526CFEF0E97A340E2E40FDE7BE0E88F5E48)
-og publiseret på [GitHub](https://github.com/swaywm/sway/releases).
+Udgivelser er signeret med [E88F5E48] og publiseret [på GitHub][GitHub
+releases].
 
 ## Installation
 
-### Fra Pakker
+### Fra pakker
 
-Sway er tilgængelig i mange distributioner. Prøv at installere pakken "sway". Hvis den ikke er tilgængelig, så tjek [denne wiki-side](https://github.com/swaywm/sway/wiki/Unsupported-packages)
-for information om installation til din(e) distribution(er).
+Sway er tilgængelig i mange distributioner. Prøv at installere "sway" pakken
+fra din.
 
-Hvis du er interesseret i at lave en Sway pakke til din distribution, burde du besøge IRC
-kanalen eller sende en e-mail til sir@cmpwn.com for rådgivning.
+Hvis du er interesseret i at pakke Sway til din distribution, tag forbi IRC
+kanalen eller send en email til sir@cmpwn.com for rådgivning.
 
 ### Kompilering fra kildekode
 
-Installation afhænger af følgende programmer:
+Se [denne wiki-side][Opsætning til udvikling] hvis du vil bygge HEAD af sway og
+wlroots til test eller udvikling.
+
+Installationsafhængigheder:
 
 * meson \*
-* [wlroots](https://github.com/swaywm/wlroots)
+* [wlroots]
 * wayland
 * wayland-protocols \*
 * pcre
 * json-c
 * pango
 * cairo
-* gdk-pixbuf2 (valgfrit tillæg: system tray)
-* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (valgfrit tillæg: man pages) \*
+* gdk-pixbuf2 (valgfrit: system tray)
+* [scdoc] (valgfrit: man pages) \*
 * git \*
 
-_\*Kompiler krav_
+_\*Kompileringsafhængighed_
 
 Kør følgende kommandoer:
 
     meson build
     ninja -C build
     sudo ninja -C build install
 
-På systemer uden 'logind', behøver du at sætte ejerens bruger-id for Sways eksekverbare filer - såkaldt SUID (Set owner User ID):
+På systemer uden logind eller seatd skal du sætte SUID på sway filen:
 
     sudo chmod a+s /usr/local/bin/sway
 
-Sway vil frasige sig 'root' tilladelser kort efter opstart
+Sway vil frasige sig 'root' tilladelser kort efter opstart.
 
 ## Konfiguration
 
-Hvis du allerede bruger i3, bør du kopiere din i3-konfiguration til `~/.config/sway/config` og
-det vil bare fungerer. Ellers skal du kopiere eksempel konfigurations filen til
-`~/.config/sway/config`. Den er normalt placeret i `/etc/sway/config`.
-Kør `man 5 sway` for at få oplysninger om konfigurationen.
+Hvis du allerede bruger i3, kopier din i3-konfiguration til
+`~/.config/sway/config` og den vil bare fungere. Ellers skal du kopiere
+eksempelkonfigurationsfilen til `~/.config/sway/config`. Den er normalt
+placeret i `/etc/sway/config`.  Kør `man 5 sway` for at få oplysninger om
+konfigurationen.
 
 ## Kører
 
-Kør `sway` fra en TTY. Nogle display managers fungerer muligvis, men understøttes ikke af
-Sway (gdm er kendt for at fungere temmelig godt).
-
+Kør `sway` fra en TTY. Nogle display managers fungerer muligvis, men hjælp ydes
+ikke fra Sway (gdm er kendt for at fungere temmelig godt).
```

Comment:
men Sway teamet kan ikke yde support.

Reply:
I updated the wording a bit here. I don't want to use "can", and right now we just use "Sway" to also mean the community in the readme's.